src/domain/accounts/domain/src/use_cases/create_account_use_case.rs

```diff
@@ -35,9 +35,13 @@ pub trait CreateAccountUseCase: Send + Sync {
 ////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
 
 #[derive(bon::Builder, Default)]
-pub struct CreateAccountUseCaseOptions {
+pub struct PredefinedAccountFields {
```

Comment:
I don't really see benefit in this rename. We will likely be implementing a Signup function sooner or later, and that would mean "create" is invoked not just for predefined accounts.

How about `AccountFields` simply?